dusty/cli/restart.py

```diff
@@ -6,21 +6,25 @@
 prior to restarting.
 
 Usage:
-  restart [--no-sync] [<services>...]
+  restart ( --repos <repos>... | [<services>...] ) [--no-sync]
 
 Options:
-  --no-sync    If provided, Dusty will not sync repos used by
-               services being restarted prior to the restart.
-  <services>   If provided, Dusty will only restart the given
-               services. Otherwise, all currently running
-               services are restarted.
+  --no-sync       If provided, Dusty will not sync repos used by
+                  services being restarted prior to the restart.
+  --repos <repos> If provided, Dusty will restart any containers
+                  that are using the repos specified.
+  <services>      If provided, Dusty will only restart the given
+                  services. Otherwise, all currently running
+                  services are restarted.
 """
 
 from docopt import docopt
 
 from ..payload import Payload
-from ..commands.run import restart_apps_or_services
+from ..commands.run import restart_apps_or_services, restart_apps_by_repo
 
 def main(argv):
     args = docopt(__doc__, argv)
+    if args['--repos']:
+        return Payload(restart_apps_by_repo, args['<repos>'], sync=not args['--no-sync'])
```

Comment:
Should abstract the common args into a dict and then include it in both payloads with **